va/x11/va_dricommon.h

```diff
@@ -83,6 +88,16 @@ struct dri_state
     void (*swapBuffer)(VADriverContextP ctx, struct dri_drawable *dri_drawable);
     union dri_buffer *(*getRenderingBuffer)(VADriverContextP ctx, struct dri_drawable *dri_drawable);
     void (*close)(VADriverContextP ctx);
+    int (*createfd)(VADriverContextP ctx, Pixmap pixmap, int *stride);
+    Pixmap
+    (*createPixmap)(VADriverContextP ctx, Drawable draw,
+                    int width, int height, int depth,
+                    int fd, int bpp, int stride, int size);
+    int
+    (*create_fence)(VADriverContextP ctx, Pixmap pixmap,
+                         struct dri3_fence *fence);
+    void (*fence_sync)(VADriverContextP ctx, struct dri3_fence *fence);
+    void (*fence_free)(VADriverContextP ctx, struct dri3_fence *fence);
 #endif
     /** \brief Reserved bytes for future use, must be zero */
     unsigned long  va_reserved[16];
```

Comment:
Please reduce the reserved bytes to keep the size of struct dri_state unchanged.

Reply:
Is it a bug that the "#ifndef ANDROID" doesn't account for a different resulting size difference when building for Android or is it accepted to have a smaller struct in that case??